asciidoc/get-started.adoc

```diff
@@ -410,9 +432,9 @@ For a comprehensive listing of all driver functionality, refer to the API docume
 {api-docs-base-uri}/javascript-driver/{javascript-driver-apidoc-version}/
 ======
 
-// [.include-with-python]
-// ======
-// {api-docs-base-uri}/python-driver/{python-driver-apidoc-version}/
-// ======
+[.include-with-python]
+======
+{api-docs-base-uri}/python-driver/{python-driver-apidoc-version}/
```

Comment:
The URL that this creates doesn't actually work - https://neo4j.com/docs/api/python-driver/4.0/ is a 404. 

The published URL appears to be https://neo4j.com/docs/api/python-driver/current/

Reply:
https://neo4j.com/docs/api/python-driver/4.0-preview/

Reply:
@AlexicaWright - is this correct? I think you mentioned removing the `4.0-preview` URL in order to create `4.0`

@martin-neotech I think the original URL was correct, we just had to update some redirects in order for the URL provided to work

Reply:
Yes, this is correct, for now it should be 4.0-preview.